.pipelines/ci/templates/build-powertoys-precheck.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     inputs:
       targetType: 'inline'
       script: |
-        $commit = "$(system.pullRequest.sourceCommitId)";
-        $gitHubCommit = Invoke-RestMethod -Method Get "https://api.github.com/repos/microsoft/PowerToys/commits/$commit"
+        $commit = "$(system.pullRequest.pullRequestNumber)";
+        $gitHubCommit = Invoke-RestMethod -Method Get "https://api.github.com/repos/microsoft/PowerToys/pulls/$commit/files"
```

Comment:
It actually needs to receive the pull number and not $commit

Reply:
Oops. I forgot to update the variable name 😅